packages/client/components/TaskWatermark.tsx

```diff
@@ -5,7 +5,8 @@ import JiraSVG from './JiraSVG'
 
 const iconLookup = {
   _xGitHubIssue: GitHubSVG,
-  JiraIssue: JiraSVG
+  JiraIssue: JiraSVG,
+  JiraServerIssue: JiraSVG
```

Comment:
I fixed the logo in next PR: https://github.com/ParabolInc/parabol/pull/6157/files#diff-d419600284cc40027d3e86f70437fc58426edbd40fe307d9d6e3dafc8a29a186R42

Reply:
I feel like for tabs in poker meetings we use an older logo. I can replace it, but not sure which logo to use. 

Here I just used the same logo, as expected there will be rarely both Jira and JiraServer integrations added. 
However, in poker meeting tabs,  2 identical logos looks weird to me.

Not sure, if there is a separate server logo exists. Maybe add some `s` badge to the logo or something?

cc @Dschoordsch

Reply:
This is the logo on our instance of Jira Server, whatcha think? 
![jira-software-logo-jira-software-logo-11562914188wp8r59nt10](https://user-images.githubusercontent.com/5514175/159831747-d78c4084-ac77-4c06-8b1e-be354b7b3e7c.png)

Reply:
@mattkrick I see this square logo both on Jira Server and Jira in the header. However, I think we can use it for Jira Server to have different logos.

Reply:
That's the logo for Jira Software, I think we should use it instead of the 3 arrows on the task card. Better version is available in the [atlassian logo library](https://atlassian.design/resources/logo-library). For the integration banner I had used the old logo they still use in their Jira Server documentation, could be the same Jira Software logo though.

Reply:
Updated the logo  
<img width="787" alt="image" src="https://user-images.githubusercontent.com/466991/160143609-c6817f37-6472-487e-a672-729b270d6569.png">